Makefile

```diff
@@ -151,8 +151,16 @@ performance_bench_report: is_docker_running
 dev_stack_run: server
 	HASH=$(hash) docker-compose -f containers/docker-compose-integration-tests.yml --project-name=integration-tests-$(hash) down
 	HASH=$(hash) docker-compose -f containers/docker-compose-dev-stack.yml --project-name=dev-stack-$(hash) down
+	HASH=$(hash) docker-compose -f containers/docker-compose-semi-dev-stack.yml --project-name=dev-stack-$(hash) down
 	HASH=$(hash) docker-compose -f containers/docker-compose-dev-stack.yml --project-name=dev-stack-$(hash) up
 
+.PHONY: semi_dev_stack_run
+semi_dev_stack_run: is_docker_running
+	HASH=$(hash) docker-compose -f containers/docker-compose-integration-tests.yml --project-name=integration-tests-$(hash) down
+	HASH=$(hash) docker-compose -f containers/docker-compose-dev-stack.yml --project-name=dev-stack-$(hash) down
```

Comment:
Why are we downing these?

Reply:
I followed the same set up for `dev stack run`. I think the logic behind it is to make sure that these containers aren't running and thus blocking ports.

Reply:
Something we'll be cleaning up with the tech-debt @Parth